components/team/TeamListItem.tsx

```diff
@@ -23,84 +51,112 @@ export default function TeamListItem(props) {
 
   return (
     team && (
-      <li className="mb-2 mt-2 divide-y">
-        <div className="flex justify-between mb-2 mt-2">
-          <div>
-            <UsersIcon className="text-gray-400 group-hover:text-gray-500 flex-shrink-0 -mt-4 mr-2 h-6 w-6 inline" />
-            <div className="inline-block -mt-1">
-              <span className="font-bold text-neutral-700 text-sm">{props.team.name}</span>
-              <span className="text-xs text-gray-400 -mt-1 block capitalize">
-                {props.team.role.toLowerCase()}
+      <li className="divide-y">
+        <div className="flex justify-between my-4">
+          <div className="flex">
+            <Avatar
+              imageSrc={
+                props.team.logo
+                  ? props.team.logo
+                  : "https://eu.ui-avatars.com/api/?background=fff&color=039be5&name=" +
+                    encodeURIComponent(props.team.name || "")
+              }
+              displayName="Team Logo"
+              className="rounded-full w-9 h-9"
+            />
+            <div className="inline-block ml-3">
+              <span className="text-sm font-bold text-neutral-700">{props.team.name}</span>
+              <span className="block -mt-1 text-xs text-gray-400">
+                {window.location.hostname}/{props.team.slug}
               </span>
             </div>
           </div>
           {props.team.role === "INVITEE" && (
             <div>
               <button
```

Comment:
Use `<Button />`

Reply:
@Jaibles is currently setting up a new design for this, hence I'm just waiting for him to finish that. Once that is done, I'll modify this bit accordingly and update the use of `<button>` or similar elements into the components we're using elsewhere.